[2022-05-31 20:28:05,337] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:28:04.475826+00:00 [queued]>
[2022-05-31 20:28:05,358] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:28:04.475826+00:00 [queued]>
[2022-05-31 20:28:05,359] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 20:28:05,360] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-05-31 20:28:05,360] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 20:28:05,376] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): run_this_first> on 2022-05-31 20:28:04.475826+00:00
[2022-05-31 20:28:05,382] {standard_task_runner.py:52} INFO - Started process 135 to run task
[2022-05-31 20:28:05,387] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'todoist_load_to_pg', 'run_this_first', 'manual__2022-05-31T20:28:04.475826+00:00', '--job-id', '383', '--raw', '--subdir', 'DAGS_FOLDER/todoist_load_to_pg.py', '--cfg-path', '/tmp/tmpsap04iw3', '--error-file', '/tmp/tmpz715svej']
[2022-05-31 20:28:05,388] {standard_task_runner.py:77} INFO - Job 383: Subtask run_this_first
[2022-05-31 20:28:05,456] {logging_mixin.py:109} INFO - Running <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:28:04.475826+00:00 [running]> on host bd658c040114
[2022-05-31 20:28:05,521] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=emil
AIRFLOW_CTX_DAG_ID=todoist_load_to_pg
AIRFLOW_CTX_TASK_ID=run_this_first
AIRFLOW_CTX_EXECUTION_DATE=2022-05-31T20:28:04.475826+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-31T20:28:04.475826+00:00
[2022-05-31 20:28:07,525] {python.py:175} INFO - Done. Returned value was: None
[2022-05-31 20:28:07,541] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=todoist_load_to_pg, task_id=run_this_first, execution_date=20220531T202804, start_date=20220531T202805, end_date=20220531T202807
[2022-05-31 20:28:07,576] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-31 20:28:07,630] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
